src/views/CVERecord/SearchResults.vue

```diff
@@ -0,0 +1,300 @@
+<template>
+  <div id="cve-secondary-page-main-container">
+    <div class="columns is-centered">
+      <div class="column is-8-desktop cve-main-column-content-width">
+        <main id="cve-main-page-content" role="main">
+          <div class="content">
+            <div v-if="cveListSearchStore.showHelpText && cveListSearchStore.totalSearchResultCount < 1">
+              <!-- cveListSearchStore.totalSearchResultCount < 1 so when there are results this doesn't flash while user is typing keyword -->
+              <p class="has-text-centered">
+                Please use the help text below the search box above to fix your keyword(s).
+              </p></div>
+            <div v-else>
+              <div v-if="cveListSearchStore.isSearching"
+                class="cve-loading-message has-text-centered"
+              >
+                <span class="icon-text">
+                  <span class="icon">
+                    <span class="loader is-loading" />
+                  </span>
+                  <span>Please wait. Loading...</span>
+                </span>
+              </div>
+              <div v-else>
+                <div v-if="cveListSearchStore.isSearchServerError" class="has-text-centered">
+                  <h1 class="title is-4">Service is currently unavailable.</h1>
+                  <p style="text-align: center;">Please
```

Comment:
I don't think this style is needed, I tried removing it and didn't see an impact. If it is needed, I would switch to the `has-text-centered` class instead